Pointzi575 paper on boarding

ChristineXYS opened the pull request.

Commit:
Fix POINTZI-575: Integrate Paper-Onboarding Framework into Pointzi SDK
Change team and bundle id.

Commit:
Fix POINTZI-575: Integrate Paper-Onboarding Framework into Pointzi SDK
Integrate changes for Pointzi.

Commit:
Fix POINTZI-575: Integrate Paper-Onboarding Framework into Pointzi SDK
Add layout for carousel.

Commit:
Fix POINTZI-575: Integrate Paper-Onboarding Framework into Pointzi SDK
Add paper onboaridng view at background, so that it doesn’t hide dismiss button.

Commit:
Fix POINTZI-575: Integrate Paper-Onboarding Framework into Pointzi SDK
Integrate showing carousel.

Commit:
Fix POINTZI-575: Integrate Paper-Onboarding Framework into Pointzi SDK
Add a outside view to show shadow, meantime inner paper-onboarding view can clip to bounds to avoid inner view go outside when corner radius applies.

Commit:
Fix IOS-1199: Preview Mode does not delete local tip feed entry when dismissed
When add StreetHawkCore_Pointzi.framework into PaperOnboarding project, it cause the duplicated header file build error again and again.
This is because SHSampleDev references StreetHawkCore.framework already, if add StreetHawkCore_Pointzi.framework there will be duplicated header files.
Test run all of them successfully:
- SHSampleDev
- StreetHawkCore
- StreetHawkCore_Pointzi
- PaperOnboarding

Commit:
Fix IOS-1199: Preview Mode does not delete local tip feed entry when dismissed
Integrate paper-onboarding clip sub view case.

Commit:
Fix IOS-1199: Preview Mode does not delete local tip feed entry when dismissed
Add notification to assign image when the carousel is already displayed.

Commit:
Fix IOS-1199: Preview Mode does not delete local tip feed entry when dismissed
Move StreetHawk header out of header.

Commit:
Fix IOS-1199: Preview Mode does not delete local tip feed entry when dismissed
Copy pointzi framework into PaperOnboarding folder to make build script pass.

Commit:
Fix IOS-1199: Preview Mode does not delete local tip feed entry when dismissed
Remove paper-onboarding project reference framework to StreetHawkCore.framework or StreetHawkCore_Pointzi.framework.
Otherwise when pod ‘pointzi/Carousel’ it links error: cannot found framework StreetHawkCore or StreetHawkCore_Pointzi.
Use “header files” referencing to pointzi header files instead, to resolve finding header file error.

Commit:
Fix IOS-1199: Preview Mode does not delete local tip feed entry when dismissed
Change header file name. It’s not in framework way but in folder file way instead.

Commit:
Fix IOS-1199: Preview Mode does not delete local tip feed entry when dismissed
Update comments.

asyncmind0 reviewed (changes requested):
Pointzi related changes should not be in paper onboarding. Only add code that is not related to pointzi in this.

Commit:
Fix POINTZI-575: Integrate Paper-Onboarding Framework into Pointzi SDK
Paper-onboarding uses dictionary to pass data, to get rid of reference to pointzi.

Commit:
Fix POINTZI-575: Integrate Paper-Onboarding Framework into Pointzi SDK
Paper-onboarding uses dictionary to pass data, to get rid of reference to pointzi.
